src/translib/transformer/xfmr_bgp_nbr.go

```diff
@@ -677,6 +773,11 @@ func fill_nbr_state_timers_info (nbr_key *_xfmr_bgp_nbr_state_key, frrNbrDataVal
         nbrTimersState.NegotiatedHoldTime = &_neg_hold_time
     }
 
+    if value, ok := frrNbrDataJson["bgpTimerKeepAliveIntervalMsecs"] ; ok {
+        _keepaliveInterval := (value.(float64))/1000
+        nbrTimersState.KeepaliveInterval = &_keepaliveInterval
+    }
+
```

Comment:
@rathnasabapathyv can you please look few missing state fields and add handling.

Reply:
Yes after reviewing that, i have seen quite few nbr/state attributes are missing.. Those are the one's that we have introduced as bgp-ext annotation post subtree transformer implemetation for nbr/state. Added support for these attributes now